batch7crud-roll10/src/main/java/com/lftechnology/batch7crud/exception/ValidationException.java

```diff
@@ -0,0 +1,29 @@
+package com.lftechnology.batch7crud.exception;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @Author Binod Shrestha <binodshrestha@lftechnology.com>
+ * Created on 1/25/16
+ */
+public class ValidationException extends Exception{
+  private Map<String, String> errors = new HashMap<>();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Make "errors" transient or serializable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1948)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this "errors" field final. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1165)